python/ray/scripts/scripts.py

```diff
@@ -554,16 +554,13 @@ def start(node_ip_address, address, port, redis_password, redis_shard_ports,
             # not provided.
             num_redis_shards = len(redis_shard_ports)
 
-        address_env = os.environ.get("RAY_REDIS_ADDRESS")
-        underlying_address = \
-            address_env if address_env is not None else address
-        if underlying_address is not None:
+        # This logic is deprecated and will be removed later.
+        if not use_gcs_for_bootstrap() and address is not None:
```

Comment:
I'm confused here. If we are in bootstraping mode, we should still setup external storage right?

Basically, use redis only for storage (kv/gcs tables)

Reply:
This is a deprecated codepath. In bootstrapping mode, external Redis is setup in `node.py`. Updated to not deprecate this yet.